src/dispatch/static/dispatch/src/incident/store.js

```diff
@@ -205,6 +205,13 @@ const actions = {
   showReportDialog({ commit }, incident) {
     commit("SET_DIALOG_REPORT", true)
     commit("SET_SELECTED", incident)
+
+    state.report.tactical.actions = incident.tasks.reduce((result, task) => {
+      if (task.status == "Resolved") {
+        return result
+      }
+      return (result ? result + "\n" : "") + "- " + task.description
+    }, "")
```

Comment:
Should we have this logic in `ReportDialog.vue` instead? We should load tasks in the actions text field in the Slack modal as well in `handle_report_tactical_command`.